plugin/linuxsty.vim

```diff
@@ -28,21 +28,21 @@ augroup linuxsty
 augroup END
 
 function s:LinuxConfigure()
-    let apply_style = 0
+    let b:apply_linux_style = 0
 
     if exists("g:linuxsty_patterns")
         let path = expand('%:p')
         for p in g:linuxsty_patterns
             if path =~ p
-                let apply_style = 1
+                let b:apply_linux_style = 1
                 break
             endif
         endfor
     else
-        let apply_style = 1
+        let b:apply_linux_style = 1
     endif
 
-    if apply_style
+    if b:apply_linux_style
```

Comment:
In fact, may I ask you to add a single line `let b:linuxsty_applied = 1` in this statement? So that we change only one line and keep consistent with an implicit *linuxsty* namespace here.

A small addition in the README file to mention this new buffer variable would be a must.
If you tell me that it works as you expect, I'll happily merge this. Thanks!